src/Directory.Build.targets

```diff
@@ -69,6 +69,10 @@
     <PackageReference Update="Microsoft.NETCore.App" PrivateAssets="All" />
   </ItemGroup>
 
+  <PropertyGroup Condition="'$(MonoBuild)' != 'true' and '$(TargetFrameworkIdentifier)' == ''">
+    <TargetFrameworkIdentifier>$([MSBuild]::GetTargetFrameworkIdentifier($(TargetFramework)))</TargetFrameworkIdentifier>
```

Comment:
Did you verify this comes before the one from D.BC.t? Also, is $(TargetFrameworkIdentifier) really not used?